simpeg/electromagnetics/static/resistivity/sources.py

```diff
@@ -139,13 +139,17 @@ class Dipole(BaseSrc):
     ----------
     receiver_list : list of simpeg.electromagnetics.static.resistivity.receivers.BaseRx
         A list of DC/IP receivers
-    location_a : (n_source, dim) numpy.array_like
-        A electrode locations; remember to set 'location_b' keyword argument to define N electrode locations.
-    location_b : (n_source, dim) numpy.array_like
-        B electrode locations; remember to set 'location_a' keyword argument to define M electrode locations.
-    location : list or tuple of length 2 of numpy.array_like
+    location_a : (dim) numpy.array_like
+        A electrode locations; remember to set 'location_b' keyword argument to
+        define N electrode locations.
+    location_b : (dim) numpy.array_like
+        B electrode locations; remember to set 'location_a' keyword argument to
+        define M electrode locations. location : list or tuple of length 2 of
+        numpy.array_like
```

Comment:
```suggestion
    location_b : (dim) numpy.array_like
        B electrode locations; remember to set 'location_a' keyword argument to
        define A electrode locations. 
    location : list or tuple of length 2 of
        numpy.array_like
```

Reply:
Nice catch. I missed that mistake. Fixing it!